### PR TITLE
FAQ - update 'bluetooth_off' entry for #971

### PR DESCRIPTION
**Pull request by heinezen**
Changelog:
- Remove "auch" from german heading
- Restructure first englis paragraph to make it more readable
- Replace "unterwegs/on the road" with "draußen/outside" to be more precise
- Include full heading in link to other FAQ entry

### CODE REVIEW

**Comment by Ein-Tim** [comment]
Should we mention that there are no risk checks carried out if Bluetooth is disabled on iOS? 

Lemme know if I should open a separate issue for this.

Text looks good 👍

File: src/data/faq_de.json [hunk 1]
```diff
@@ -551,12 +551,13 @@
...
                         "textblock": [
-                            "Dann können Begegnungen mit den anderen Smartphones, die die Corona-Warn-App laufen haben, nicht aufgezeichnet werden, weil keine Zufalls-IDs ausgetauscht werden. Haben Sie deswegen bitte immer Bluetooth und bei Android auch die Standort-Dienste aktiviert, wenn Sie in Kontakt mit anderen Leuten sind, zum Beispiel, wenn Sie Gäste bekommen oder Sie unterwegs sind.",
-                            "Bei Android müssen die Standort-Dienste nur aktiviert sein, die Corona-Warn-App verwendet den Standort nicht. Für alle anderen Apps können Sie die Standortverwendung ausschalten, wenn Sie das möchten: siehe <a href='#android_location' target='_blank' rel='noopener'>Meldungen zur Aktivierung der Standortverwendung</a>."
+                            "Dann können Begegnungen mit den anderen Geräten, die die Corona-Warn-App laufen haben, nicht aufgezeichnet werden, weil keine Zufalls-IDs ausgetauscht werden. Haben Sie deswegen bitte immer Bluetooth (und bei Android auch die Standort-Dienste) aktiviert, wenn Sie in Kontakt mit anderen Leuten sind, zum Beispiel, wenn Sie Gäste haben oder nach draußen gehen.",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:558]: Perhaps instead of:
"Haben Sie deswegen ..."
alternative
"Halten Sie deswegen ..." ?

**Review by MikeMcC399 — COMMENTED** [review]
One tentative suggestion.

**Comment by heinezen** [comment]
> Should we mention that there are no risk checks carried out if Bluetooth is disabled on iOS?

@Ein-Tim It became two complicated to keep all the specifics in one entry, so I split them into iOS/Android-specific ones.

@MikeMcC399 I rephrased to sentence to make it sound more active in tone.

**Comment by heinezen** [comment]
@dsarkar @loki-cui Please check if changing anchors affects the app or templates for the store.

**Comment by MikeMcC399** [comment]
@heinezen 
> I rephrased to sentence to make it sound more active in tone.

Google FAQs (English & German) `#bluetooth_off_android` look good to me!

**Comment by Ein-Tim** [comment]
The iOS FAQ also looks good to me!

File: src/data/faq_de.json (hunk 1 above)
Comment (fynngodau) [reply src/data/faq_de.json]: "Halten Sie Bluetooth aktiviert" and "Haben Sie Bluetooth aktiviert" both don't sound great to me.

Suggestions:
* "Aktivieren Sie Bluetooth …"
* "Lassen Sie Bluetooth aktiviert …"
* "Stellen Sie sicher, dass Bluetooth immer aktiviert ist, wenn …"

Comment (MikeMcC399) [reply src/data/faq_de.json]: @fynngodau 
The above is an outdated comment. Check instead https://github.com/corona-warn-app/cwa-website/pull/1052/files and the line says

"Aktivieren Sie deswegen bitte immer Bluetooth (und bei Android bis zur Version 10 auch die Standort-Dienste), wenn Sie in Kontakt mit anderen Leuten sind, zum Beispiel, wenn Sie Gäste haben oder nach draußen gehen."

Comment (fynngodau) [reply src/data/faq_de.json]: Ah, sorry. I think you can mark this as resolved if it's fixed, can't you?

Comment (MikeMcC399) [reply src/data/faq_de.json]: I don't have the option to mark as resolved here. Sorry.

**Pull request merged**